src/tasks/daily/misc/daily_logistics_mixin.py

```
@@ -107,26 +132,35 @@ def delivery_send_others(self):
                 for i in range(start_index, len(steps)):
                     step = steps[i]
                     feature = step[0]
                     box = step[1]
                     after_sleep = step[2]
                     time_out = 12 if i > 2 else 5
+                    if i in blind_spot_steps:
+                        self.log_info("盲点加速：装货过场动画期间快速连点屏幕中心")
+                        if not self.blind_spot_speed_up(feature=feature, box=box, time_out=time_out):
+                            if i in optional_steps:
+                                continue
+                            self.log_info(f"步骤 {feature} 未找到，跳过本次活动")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)